src/components/molecules/Monaco/Monaco.tsx

```diff
@@ -257,12 +288,28 @@ const Monaco = (props: {editorHeight: string}) => {
 
   // read-only if we're in preview mode and another resource is selected - or if nothing is selected at all
   const isReadOnlyMode = useCallback(() => {
-    return (
-      (isInPreviewMode && selectedResourceId !== previewResourceId) ||
-      selectedValuesFileId !== previewValuesFileId ||
-      (!selectedPath && !selectedResourceId)
-    );
-  }, [isInPreviewMode, selectedResourceId, previewResourceId, selectedValuesFileId, previewValuesFileId, selectedPath]);
+    if (isInPreviewMode && previewType === 'cluster') {
+      return false;
+    }
+    if (isInPreviewMode && selectedResourceId !== previewResourceId) {
+      return false;
```

Comment:
I think these should all be true and the default false

Reply:
Right, will update it